### ISSUE TITLE
When navigating to other dashboard, the header bar becomes partially visible

### ISSUE THREAD

**Issue by Rijswijker** [thomasloven/hass-browser_mod#668]
My Home Assistant version: 2023.12

**What I am doing:**
When i do a Navigate action within the same dashboard (to subview) everything is working as expected.
But when I navigate to another dashboard or a subview on another dashboard the header bar becomes partially visible.

**Tap Action:** `Navigate`
**Navigation Path:** `/another-dashbord/subview`

![titlebar](https://github.com/thomasloven/hass-browser_mod/assets/4473594/d93929d8-47ad-4bb3-a095-bcf64bd9e0cf)

The header bar only goes away again when I am on the default dashboard and change subview there to the default dashboard.
If I change subviews on the non-default dashboard, the header bar remains.

By "default dashboard", I mean the dashboard that is defined in the settings of browser-mod in the Frontend Settings "Default dashboard" section.

**What I expected to happen:**
That the header is not shown.

**What happened instead:**
A part of the header is shown.

**Minimal** steps to reproduce:
Change to another dashboard (or subview on another dashboard) when the setting `hide header` is set to `true`.

```yaml
# The least amount of code or steps possible to reproduce my error

# Put your code/steps here
tap_action:
  action: navigate
  navigation_path: /another-dashboard-than-default/subview-on-other-dashboard
```

**Error messages from the browser console:**

```
// Select everything from the browser console and copy it

No errors visible
```

---

**By replacing the space in the checkboxes ([ ]) with an X below, I indicate that I:**

- [X] Understand that this is a channel for reporting bugs, not a support forum (https://community.home-assistant.io/).

- [X] Have made sure I am using the latest version of the plugin.

- [X] Have followed the troubleshooting steps of the "Common Problems" section of https://github.com/thomasloven/hass-config/wiki/Lovelace-Plugins.

- [X] Understand that leaving one or more boxes unticked or failure to follow the template above may increase the time required to handle my bug-report, or cause it to be closed without further action.

**Comment by ViRuSTriNiTy** [comment]
Yep, also happens when the user opens the more-info dialog, then clicks `Show more` followed by the back button of the browser:

![image](https://github.com/thomasloven/hass-browser_mod/assets/12170676/36a88482-bdad-4396-82ff-cb855cebd5ac)

**Comment by Flintr** [comment]
Same here. Any workarounds?
